src/json-utils.js

```diff
@@ -50,3 +50,52 @@ function getAttributes(entity) {
   }
   return elemObj;
 }
+
+function createEntities(EntitiesJSON) {
+  const entitiesData = EntitiesJSON.data;
+
+  // trick to create nodeList with entities to return as result
+  const entityElements = document.createElement('a-entity');
+
+  for (let entityData of entitiesData) {
+    entityElements.appendChild(createEntity(entityData))
+  }
+
+  return entityElements;
+}
+
+/*
+Add a new entity with a list of components and children (if exists)
+ * @param {object} entityData Entity definition to add:
+ *   {element: 'a-entity', components: {geometry: 'primitive:box'}}
+ * @param {Element} parentEl the parent element to which the Entity will be added
+ * @return {Element} Entity created
+*/
+function createEntity(entityData, parentEl) {
+  const entity = document.createElement(entityData.element);
+
+  // load default attributes
+  for (let attr in entityData.components) {
+    entity.setAttribute(attr, entityData.components[attr]);
+  }
+
+  if (entityData.id) {
+    entity.setAttribute("id", entityData.id);
+  }
+  if (entityData.class) {
+    entity.setAttribute("className", entityData.class);
```

Comment:
This is not React here, it's vanilla js. You need to use `class` not `className`.
`entity.setAttribute("class", entityData.class);`

Reply:
same issue in the saving part https://github.com/3DStreet/3dstreet/blob/fd568a7cd78d89c6087a365f55069d5303cea113/src/json-utils.js#L39-L41

Reply:
My bad, we indeed have `entity.className` in vanilla js, so that's ok for the saving part. But `entity.setAttribute('className', 'something')` is definitely not setting the class, `entity.className` or `entity.getAttribute('class')` don't change.